### ISSUE TITLE
Introductory Tutorial: How to Play the Game of Consequences

### ISSUE THREAD

**Issue by vianneyroux** [mdbesten/what-could-go-wrong#78]
One improvement to ease newcomers into presenting the game could be to provide a brief overview or tutorial before diving into the gameplay. Here's an example of what the amateur presenter could say:

"Before we start, let me quickly guide you through how to play. Each player will draw 5 white cards from their deck. We'll spin to select the first Card Czar, who will then draw a black prompt card and read it aloud. Everyone else will choose one white response card and place it face down. The Card Czar will read each response aloud and pick one to discuss further, awarding a point to the player whose card is chosen. We'll discuss possible outcomes based on the chosen card, with points awarded for good points made. Remember, some cards might be sensitive, so let's be respectful and open-minded in our discussions. Ready to dive in?"

**Comment by mdbesten** [comment]
Reminds me of issue #58

**Comment by Farushk** [comment]
I completely agree that a brief tutorial would be beneficial for newcomers! To add on to your great suggestion, maybe the intro could also mention the game's purpose - exploring the potential risks of LLMs.  We could briefly explain the two card types (prompts and responses) and the point system for winning cards and making good discussion points. This would set expectations and get everyone comfortable before diving in!
